src/Analysis/Ast/Impl/Modules/PythonModule.cs

```diff
@@ -243,6 +243,24 @@ internal PythonModule(ModuleCreationOptions creationOptions, IServiceContainer s
             }
         }
 
+        /// <summary>
+        /// Returns the string line corresponding to the given location
+        /// </summary>
+        /// <param name="line">The line number</param>
+        public string GetLine(int line) {
```

Comment:
I'd be more inclined to make this an extension method in PythonModuleExtensions, rather than making it actually on the type.

Reply:
Changed.